drafts/signals-everywhere.md

```diff
@@ -0,0 +1,162 @@
+Those are implementation details and don't change what's happening underneath.
+
+These source Signals can be used to construct new ones, building a reactive chain of Signals.
+**The secret key to all of this are the side effects though.**
+Without them, nothing would ever happen.
+Just having a reactive value doesn't _do_ anything.
+It's just hanging there until someone asks for its value.
+That's exactly what effects do.
+They make the (Signals) world go round.
+Every time one of the sources changes, it produces a side effect.
+This could be fetching data from an API or … updating the UI.
+
+### Pure Signals Magic
+
+If you think about your declarative view template as an effect of the Signals used inside, the pieces fall in place.
+It's – ironically – the essence of React's `UI = fn(state)`.
+Let's rephrase it as **`UI = effect(signal)`** and it's obvious.
+We could strip away all the comfort of UI frameworks and limit ourselves to Signals, building our reactive UI.
+The following example uses the Preact Signals core library, which can be used without Preact or any other dependencies.
+
+```html
+<html>
+  <body></body>
+  <script type="module">
+    import { signal, effect } from "https://cdn.jsdelivr.net/npm/@preact/signals-core@1.5.1/+esm";
+
+    const count = signal(0);
+
+    const button = document.createElement("button");
+    button.innerText = "click me";
+    const paragraph = document.createElement("p");
+    document.body.append(button, paragraph);
+
+    button.addEventListener("click", () => count.value += 1);
+
+    effect(() => (paragraph.innerText = count.value));
+  </script>
+</html>
+```
+
+This is powerful stuff.
+We use plain JavaScript to create our elements and leverage Signals to keep them up-to-date with the state.
+The outcome of the above example is pretty much the same as the [official Preact Signals example](https://preactjs.com/guide/v10/signals#introduction).
+
+### Conclusion and Outlook
+
+With reactivity "solved", the core concern of JavaScript UI frameworks going forward will be declarative templating.
+The main purpose "keeping the UI in sync with the state" **holds no longer true** for them.
+It will be more about ergonomics, developer experience, offering higher-level features (e.g. routing) and integration with the server.
+Those aspects have already been the drivers for the so-called meta frameworks like Next.js, Remix, Nuxt and SvelteKit.
+I predict that their underlying frameworks get less important over time.
+Choosing for example Svelte over Angular is mostly going to be based on preferences.
+Alternatively, you decide on a meta-framework, which has made the choice for you.
+
+Web development is as exciting as ever.
+How do you think will Signals influence the future of building for the web?
+Am I overrating them?
+What does this mean for React's long-running dominance?
```

Comment:
I would probably add something like: "Let me know your thoughts in the comments" or something like that. Otherwise it's a bit a dangling end to the article.